ev3dev/core.py

```diff
@@ -893,7 +893,7 @@ class FirgelliL1250Motor(Motor):
 
     def __init__(self, address=None, name_pattern=SYSTEM_DEVICE_NAME_CONVENTION, name_exact=False, **kwargs):
 
-        super(FirgelliL1250Motor, self).__init__(address, name_pattern, name_exact, **kwargs)
+        super(FirgelliL1250Motor, self).__init__(address, name_pattern, name_exact, driver_name=['fi-l12-ev3-50'], **kwargs)
```

Comment:
This should be the new driver name (the company is now called Actuonix, or something like that); we might need to update the spec for that. The class name needs to be renamed as well.